### PR TITLE
REFAC(client): replace NULL and Q_NULLPTR with nullptr

### PR DESCRIPTION
**Pull request by Popkornium18**
This changes all occurances of NULL and Q_NULLPTR in the mumble source dir to nullptr. Additionally comparisons with NULL were changed to `if(pointer)` if it would not be confusing. (e.g. booleans as parameters `setEnabled(pointer != nullptr)`).

I plan to do the rest of the code, however it is probably best not to do everything at once.

Obviously I couldn't check everything for regressions. It compiles and I did not experience any problems though.

Implements #3763 (partly)

### CODE REVIEW

File: src/mumble/ACLEditor.cpp [hunk 1]
```diff
@@ -592,7 +592,7 @@ void ACLEditor::groupEnableCheck() {
 	qpbGroupRemoveRemove->setEnabled(enabled);
 	qpbGroupInheritRemove->setEnabled(enabled);
 
-	enabled = (gp != NULL);
+	enabled = (gp);
```
Comment (Krzmbrzl) [inline src/mumble/ACLEditor.cpp:595]:
```suggestion
	enabled = gp;
```

**Comment by Krzmbrzl** [comment]
Just a small thingy: You have a typo in your commit message:
"...w**h**ere replaced" - see https://www.differencebetween.com/difference-between-where-and-vs-were/ :D 

So as I understand it, this PR replaces all NULL-stuff in the client-code, correct?

**Comment by Popkornium18** [comment]
Made the changes you requested

**Comment by Krzmbrzl** [comment]
As always: Thanks for your effort :D

**Comment by davidebeatrici** [comment]
Found a bug:
https://github.com/mumble-voip/mumble/blob/0ba1eac51b5f4443e6fbb1bc3e157403903e9a01/src/mumble/MumbleApplication.cpp#L71-L74

**Comment by Krzmbrzl** [comment]
@davidebeatrici any reason why you didn't simply fix this one? :thinking:

**Comment by davidebeatrici** [comment]
@ZeroAbility and I found it randomly. The client was crashing for him in `injectMouseMessage()`.

The correct code is:
```cpp
if (!gsw) {
 	return false;
}
```

**Comment by Krzmbrzl** [comment]
Yeah I know what the correct code is. What I don't understand is why you post it here instead of creating a PR to fix it? :thinking:

**Comment by davidebeatrici** [comment]
I wanted to keep track of it while looking for other typos (didn't find any) before creating the PR.

**Pull request merged**